### DISCUSSION TITLE
Live Export Data? - Home Mini

### DISCUSSION THREAD

**Discussion by owenashurst** [BottlecapDave/HomeAssistant-OctopusEnergy#940] [Q&A]
Hi,

I recently got my Export Tariff the other day and finally started showing yesterday on my account. I have a Home Mini also for tracking my live usage.

I've noticed in HA my Export meter has been added as a third device (Import Electric, Export Electric and Gas). In the export meter I can see I have the following entities:

![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/4403436/23520baf-28b2-4cad-8c0a-52d3b052752a)

I've noticed however, there is no current export usage. Only previous accumulative usage and cost.

I understand from another thread that you don't have an export tariff @BottlecapDave, however I'm happy to invoke the API which you think it would be in if so. I've tried the GraphQL API, however I'm unsure where to get my deviceId from for the following query:

```
query {
	smartMeterTelemetry(
    deviceId: ""
	) {
    readAt
    consumption
		consumptionDelta
    demand
	}
}
```

EDIT: I found my deviceId by enabling debugging on the Octopus Addon in HA. With that I can see the consumption is either positive when importing and negative when exporting. I'm guessing this means that Octopus's API doesn't have any up to date information on exporting 😢 

Output from the above GraphQL query:

```
{
    "data": {
        "smartMeterTelemetry": [
            {
                "readAt": "2024-06-30T14:39:06+00:00",
                "consumption": "7144982",
                "consumptionDelta": null,
                "demand": "-101"
            }
        ]
    }
}
```

It's worth noting the `deviceId` is the same within both the `smartImportElectricityMeter` and `smartExportElectricityMeter` objects - which makes sense since it's a SMETS2 meter that records both importing/exporting.

Thanks,
Owen

**Comment by BottlecapDave** [comment]
Hello. I'm afraid your findings is correct, OE doesn't provide export information via the Home Mini. I specify this in the [docs](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/entities/electricity/?h=export#home-mini-entities) of the various entities, but will also add to the other areas the Home Mini is mentioned.

**Reply by deltamelter** [reply]
Hi @BottlecapDave ,
I added a small solar setup and saw that my meter stores both import and export.  Today I noticed that the smartMeterTelemetry returns an the exported total and this was updating in "realtime" so I assume was collected by the mini.  I will never qualify for an export MPAN so would be useful if this  " importMeter" export was accessible in the HA integration 
Also, do you know where I might find this in longer term meter readings, because smartMeterTelemetry doesn't seem to go back to the beginning of my exporting at the end of march.  

```
query live ($device_id: String!, $start: DateTime!, $end: DateTime!  ){
	smartMeterTelemetry(
    deviceId: $device_id
    grouping: HALF_HOURLY 
    start: $start
    end: $end
	) {
    		readAt
		consumptionDelta
		costDeltaWithTax
    		demand
    		export
	}
}
```
```
         {
            "readAt" : "2025-04-24T13:15:00+00:00",
            "consumptionDelta" : "12.0000",
            "costDeltaWithTax" : "0.1957",
            "demand" : "702.0",
            "export" : "3503.0"
         },
         {
            "readAt" : "2025-04-24T13:16:00+00:00",
            "consumptionDelta" : "13.0000",
            "costDeltaWithTax" : "0.2120",
            "demand" : "681.0",
            "export" : "3503.0"
         },
         {
            "readAt" : "2025-04-24T13:17:00+00:00",
            "consumptionDelta" : "3.0000",
            "costDeltaWithTax" : "0.0489",
            "demand" : "0.0",
            "export" : "3504.0"
         },
         {
            "readAt" : "2025-04-24T13:18:00+00:00",
            "consumptionDelta" : "0",
            "costDeltaWithTax" : "0.00000",
            "demand" : "0.0",
            "export" : "3506.0"
         },
         {
            "readAt" : "2025-04-24T13:19:00+00:00",
            "consumptionDelta" : "0",
            "costDeltaWithTax" : "0.00000",
            "demand" : "0.0",
            "export" : "3508.0"
         },
         {
            "readAt" : "2025-04-24T13:20:00+00:00",
            "consumptionDelta" : "0",
            "costDeltaWithTax" : "0.00000",
            "demand" : "0.0",
            "export" : "3509.0"
         },
         {
            "readAt" : "2025-04-24T13:21:00+00:00",
            "consumptionDelta" : "0",
            "costDeltaWithTax" : "0.00000",
            "demand" : "0.0",
            "export" : "3511.0"
         },
```

**Reply by BottlecapDave** [reply]
Interesting, they must have added that recently although I can't find it referenced in their changelog. I'll try and add it in a future release. It would be interesting to know if the same device id is being reported in your account settings to I can link the meters correctly. This can be determined by turning debug mode on and then reloading the integration. You should get a debug log within an hour stating the raw account response.

**Reply by deltamelter** [reply]
Yes, the device_ID is the same in the logs as used in the above query.  I fetched it with readyAPI with `getAccountPropertiesMeters`
"device_id" : "${getAccountPropertiesMeters#Response#$['data']['account']['properties'][0]['smartDeviceNetworks'][0]['smartDevices'][1]['deviceId']}",
```
query getAccountPropertiesMeters($accountNumber: String!) {
  account(accountNumber: $accountNumber) {
    properties {
      smartDeviceNetworks {
        smartDevices {
        	id
          deviceId
          type
        }
      }
    }
  }
}
```

in the format "00-23-7E-XX-XX-XX-XX-XX"

**Reply by BottlecapDave** [reply]
The integration works a little differentoy in that it associates the mini data with a physical meter. This is done by looking at the smart import/export meter properties of the meters https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/b3342664e9781f81f1826b6356153e73d5219924/custom_components/octopus_energy/api_client/__init__.py#L38. it would be interesting to know if the import and export meter both report the same smart meter id (I believe they will)

**Reply by deltamelter** [reply]
I don't have an export meter so export meter details are `null` for me.
The same query in the integration returns the same deviceID as I was retrieving with my query..  
```
{
   "data" : {
      "octoplusAccountInfo" : {
         "isOctoplusEnrolled" : true
      },
      "octoHeatPumpControllerEuids" : [ ],
      "account" : {
         "electricityAgreements" : [
            {
               "meterPoint" : {
                  "mpan" : "2000XXXXXXXXX",
                  "meters" : [
                     {
                        "activeFrom" : "2021-10-26",
                        "activeTo" : null,
                        "makeAndType" : "ELSTER AS302P S2 PP",
                        "serialNumber" : "XXXXXXXXXXX",
                        "meterType" : "S2ADE",
                        "smartExportElectricityMeter" : null,
                        "smartImportElectricityMeter" : {
                           "deviceId" : "00-23-XX-XX-XX-XX-XX-XX",
                           "manufacturer" : "1057 - Honeywell Elster",
                           "model" : "00000201",
                           "firmwareVersion" : "09024011"
                        }
                     }
                  ],
```

**Reply by BottlecapDave** [reply]
I guess the issue in this case is then there's nothing to say if an export sensor should be supplied, as I don't have any solar array so wouldn't want to see the sensor. I'll need to think how this is exposed. It could be provided in a disabled state, but would be a bit weird being associated with the import meter

**Reply by theneilkirk** [reply]
According to their API docs, export is available (now) using the export MPAN and serial and although it returns as 'consumption', it is export:
[https://developer.octopus.energy/rest/guides/endpoints#api-consumption-endpoints](https://developer.octopus.energy/rest/guides/endpoints#api-consumption-endpoints)

The following REST query returns my export results:
`https://api.octopus.energy/v1/electricity-meter-points/{{export mpan}}/meters/{{export serial}}/consumption/?period_from=2025-05-15T00:00:00Z&period_to=2025-05-15T23:00:00Z`

```
{
    "count": 47,
    "next": null,
    "previous": null,
    "results": [
        {
            "consumption": 0.0,
            "interval_start": "2025-05-16T00:00:00+01:00",
            "interval_end": "2025-05-16T00:30:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T23:30:00+01:00",
            "interval_end": "2025-05-16T00:00:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T23:00:00+01:00",
            "interval_end": "2025-05-15T23:30:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T22:30:00+01:00",
            "interval_end": "2025-05-15T23:00:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T22:00:00+01:00",
            "interval_end": "2025-05-15T22:30:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T21:30:00+01:00",
            "interval_end": "2025-05-15T22:00:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T21:00:00+01:00",
            "interval_end": "2025-05-15T21:30:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T20:30:00+01:00",
            "interval_end": "2025-05-15T21:00:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T20:00:00+01:00",
            "interval_end": "2025-05-15T20:30:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T19:30:00+01:00",
            "interval_end": "2025-05-15T20:00:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T19:00:00+01:00",
            "interval_end": "2025-05-15T19:30:00+01:00"
        },
        {
            "consumption": 0.001,
            "interval_start": "2025-05-15T18:30:00+01:00",
            "interval_end": "2025-05-15T19:00:00+01:00"
        },
        {
            "consumption": 0.052,
            "interval_start": "2025-05-15T18:00:00+01:00",
            "interval_end": "2025-05-15T18:30:00+01:00"
        },
        {
            "consumption": 0.117,
            "interval_start": "2025-05-15T17:30:00+01:00",
            "interval_end": "2025-05-15T18:00:00+01:00"
        },
        {
            "consumption": 0.097,
            "interval_start": "2025-05-15T17:00:00+01:00",
            "interval_end": "2025-05-15T17:30:00+01:00"
        },
        {
            "consumption": 0.162,
            "interval_start": "2025-05-15T16:30:00+01:00",
            "interval_end": "2025-05-15T17:00:00+01:00"
        },
        {
            "consumption": 0.173,
            "interval_start": "2025-05-15T16:00:00+01:00",
            "interval_end": "2025-05-15T16:30:00+01:00"
        },
        {
            "consumption": 0.243,
            "interval_start": "2025-05-15T15:30:00+01:00",
            "interval_end": "2025-05-15T16:00:00+01:00"
        },
        {
            "consumption": 0.337,
            "interval_start": "2025-05-15T15:00:00+01:00",
            "interval_end": "2025-05-15T15:30:00+01:00"
        },
        {
            "consumption": 0.224,
            "interval_start": "2025-05-15T14:30:00+01:00",
            "interval_end": "2025-05-15T15:00:00+01:00"
        },
        {
            "consumption": 0.178,
            "interval_start": "2025-05-15T14:00:00+01:00",
            "interval_end": "2025-05-15T14:30:00+01:00"
        },
        {
            "consumption": 0.29,
            "interval_start": "2025-05-15T13:30:00+01:00",
            "interval_end": "2025-05-15T14:00:00+01:00"
        },
        {
            "consumption": 0.157,
            "interval_start": "2025-05-15T13:00:00+01:00",
            "interval_end": "2025-05-15T13:30:00+01:00"
        },
        {
            "consumption": 0.053,
            "interval_start": "2025-05-15T12:30:00+01:00",
            "interval_end": "2025-05-15T13:00:00+01:00"
        },
        {
            "consumption": 0.199,
            "interval_start": "2025-05-15T12:00:00+01:00",
            "interval_end": "2025-05-15T12:30:00+01:00"
        },
        {
            "consumption": 0.152,
            "interval_start": "2025-05-15T11:30:00+01:00",
            "interval_end": "2025-05-15T12:00:00+01:00"
        },
        {
            "consumption": 0.088,
            "interval_start": "2025-05-15T11:00:00+01:00",
            "interval_end": "2025-05-15T11:30:00+01:00"
        },
        {
            "consumption": 0.144,
            "interval_start": "2025-05-15T10:30:00+01:00",
            "interval_end": "2025-05-15T11:00:00+01:00"
        },
        {
            "consumption": 0.162,
            "interval_start": "2025-05-15T10:00:00+01:00",
            "interval_end": "2025-05-15T10:30:00+01:00"
        },
        {
            "consumption": 0.239,
            "interval_start": "2025-05-15T09:30:00+01:00",
            "interval_end": "2025-05-15T10:00:00+01:00"
        },
        {
            "consumption": 0.208,
            "interval_start": "2025-05-15T09:00:00+01:00",
            "interval_end": "2025-05-15T09:30:00+01:00"
        },
        {
            "consumption": 0.179,
            "interval_start": "2025-05-15T08:30:00+01:00",
            "interval_end": "2025-05-15T09:00:00+01:00"
        },
        {
            "consumption": 0.037,
            "interval_start": "2025-05-15T08:00:00+01:00",
            "interval_end": "2025-05-15T08:30:00+01:00"
        },
        {
            "consumption": 0.014,
            "interval_start": "2025-05-15T07:30:00+01:00",
            "interval_end": "2025-05-15T08:00:00+01:00"
        },
        {
            "consumption": 0.026,
            "interval_start": "2025-05-15T07:00:00+01:00",
            "interval_end": "2025-05-15T07:30:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T06:30:00+01:00",
            "interval_end": "2025-05-15T07:00:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T06:00:00+01:00",
            "interval_end": "2025-05-15T06:30:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T05:30:00+01:00",
            "interval_end": "2025-05-15T06:00:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T05:00:00+01:00",
            "interval_end": "2025-05-15T05:30:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T04:30:00+01:00",
            "interval_end": "2025-05-15T05:00:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T04:00:00+01:00",
            "interval_end": "2025-05-15T04:30:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T03:30:00+01:00",
            "interval_end": "2025-05-15T04:00:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T03:00:00+01:00",
            "interval_end": "2025-05-15T03:30:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T02:30:00+01:00",
            "interval_end": "2025-05-15T03:00:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T02:00:00+01:00",
            "interval_end": "2025-05-15T02:30:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T01:30:00+01:00",
            "interval_end": "2025-05-15T02:00:00+01:00"
        },
        {
            "consumption": 0.0,
            "interval_start": "2025-05-15T01:00:00+01:00",
            "interval_end": "2025-05-15T01:30:00+01:00"
        }
    ]
}
```

**Reply by BottlecapDave** [reply]
Export sensors are already available using this API as this API is based on historic data from the meter. Octopus mini live export data is new, and in my backlog to look at. The "issue" is the smart device is associated with an import meter, so providing export data via this meter would be different to all other sensors and would be a little weird.

**Reply by theneilkirk** [reply]
Thanks for explaining. I think I get it now.

I didn't know there were export sensors for this data though? I've been trying to find some for the _Return to grid_ settings in the Energy Dashboard.

**Reply by Dukeicon** [reply]
@theneilkirk To view the export data in the Energy Dashboard, you need to use the Previous Accumulative 'statistic entities'.

Note that this isn't live data and gets added to the HA Statistic Entity the following day usually some time in the afternoon (20th May has just come in in the last hour or so for me).
Once this has imported it updates the Energy Dashboard and will show negative export values in the 'Energy Usage' section.

On the Energy Dashboard > 2 dots top right > Energy Configuration.
Edit the 'Return to Grid' part and add the following two statistic entities.

- Previous Accumulative Consumption Export Electricity (XXXXXXXXXX/XXXXXXXXXXXXX)
- Previous Accumulative Cost Export Electricity (XXXXXXXXXX/XXXXXXXXXXXXX)

There's a bit about [adding it in the Docs](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/setup/energy_dashboard/#previous-day-consumption), although it just refers to Electricty used rather than exported but is the same principle.


**Graph with Live Export from Mini, Live Solar Production (from Inverter) and waiting for 'Previous Accumulative' Export data.**
<img width="940" alt="Without Previous Accumulative" src="https://github.com/user-attachments/assets/034f8b14-82b3-4cd8-bae0-514700111ba3" />

**Graph with Live Export from Mini, Live Solar Production (from Inverter) and 'Previous Accumulative' Export data.**
<img width="941" alt="With Previous Accumulative" src="https://github.com/user-attachments/assets/ec1be000-f64e-4bcc-8467-8827629447f8" />

**Reply by Dukeicon** [reply]
> Export sensors are already available using this API as this API is based on historic data from the meter. Octopus mini live export data is new, and in my backlog to look at. The "issue" is the smart device is associated with an import meter, so providing export data via this meter would be different to all other sensors and would be a little weird.

Hi @BottlecapDave - I have a Mini, Solar & on an Export Tariff. Happy to help with testing or running queries if you need it.

**Comment by BottlecapDave** [comment]
This has been implemented as part of [v16.0.0](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/releases/tag/v16.0.0)

**Reply by deltamelter** [reply]
Thanks Dave
<img width="1117" height="777" alt="image" src="https://github.com/user-attachments/assets/65e3d1de-0171-486b-8467-8de0aaeb0f99" />

**Reply by stephencrowther** [reply]
So should there be live export data available now via the home mini has octopus changes this? I can’t see a ‘Current Total Export Electricity’ entity in my setup.

**Reply by stephencrowther** [reply]
Ignore me I’ve read some other threads and found an answer - it’s in the import meter and disabled by default. I have a Kaifa 5 terminal meter and it returns u known unfortunately.

**Reply by MPC-GH** [reply]
This has been working great for me through September until 11.45 27th September, after which 'Current Total Export Electricity' has started returning unknown.  I'm not aware of something changing in my HA yesterday so I'm guessing this is something changing on Octopus' end, either with my account or more generally.

Anyone else started seeing this on their account?

I've been tracking HA 2025.9 point releases and I'm on 2025.9.4.  I have a Mini, but the panels (since 2015) export under deemed FIT rates rather than to an octopus export tariff.  I had a home battery installation at the end of August and don't have a dedicated export meter.

<img width="718" height="417" alt="image" src="https://github.com/user-attachments/assets/1ab937fc-17e7-45f8-90b1-d18863efc3d6" />

**Reply by BottlecapDave** [reply]
If something works and then stops working, it's usually an API issue. I think I saw in the forums something similar (but that might be you). In the past a reboot of the home mini has also sometimes kicked it.

**Reply by MPC-GH** [reply]
Thanks Dave.  I'm relatively new to the integration as my smart meters were only fitted in August.  This integration's made things so easy out of the box!

It wasn't me in the forums though.  I did a quick google looking for similar issues to mine here and didn't find a link to them, but did find this thread.  Are the forums open to all?

Is there a quick guide to troubleshooting API issues or recommendations for the best way to approach octopus's support?  I do have an IT background but it's been a while since I was actively coding so I'm a bit rusty at this!

I'm working my way through the FAQ and will open an issue with logs if a reboot of the mini doesn't help.

Greatly appreciate the feedback.

**Reply by BottlecapDave** [reply]
The sensor just reports what the API reports back. As it doesn't seem widespread, it's probably account related. If you download the diagnostics from the integration (details in faq), then this will grab the latest value (if available) from the API directly.

The forums (forums.octopus.energy) is closed to my knowledge, but I think you can request access. The person in the forum last message was they contacted hello@ and awaiting a response https://forum.octopus.energy/t/api-returns-0-for-export/12988/4

**Reply by deltamelter** [reply]
I was going to reply yesterday that all was well and working correctly, but it looks like my export meter reading was reset last night...
<img width="749" height="278" alt="image" src="https://github.com/user-attachments/assets/320bd9b3-5c7d-4e08-ae4d-ca5b4018e45a" />
Still 0 now despite exporting all afternoon.

```
            "readAt" : "2025-09-30T13:00:00+00:00",
            "consumptionDelta" : "30.0000",
            "costDeltaWithTax" : "0.4366",
            "demand" : "0.0",
            "export" : "0.0"
```

**Reply by me-o-c** [reply]
Just wanted to add that I'm experiencing the same issue with "Current Total Export Electricity", last reading was returned 27th September 15:39:57. I will request access to the Octopus forum and check the link Dave posted above.

**Reply by MPC-GH** [reply]
I grabbed the diagnostics and it does show last update/change was when the sensor stopped updating.  I guess it would have been useful to grab these before it stopped working for a comparison, but I don't have access to a time machine.  I do know that I have a query open with hello@ regarding my IHD gas display not working so I suspect something was twiddled that's caused the export data on the electricity side to stop being gathered/forwarded back.

I did power off and on the Octo-Mini but that hasn't made a difference.

```
      "octopus_energy_electricity_1_2_current_total_export": {
        "state": "unknown",
        "attributes": {
          "is_export": false,
          "is_smart_meter": true
        },
        "last_updated": "2025-09-29T01:34:18.510999+00:00",
        "last_changed": "2025-09-29T01:34:18.510999+00:00"
      },
```
Looking at the electricity meter, there's no indication of exported power showing in there.  As mentioned above, I don't have a separate export meter.

```
      "electricity_meter_points": [
        {
          "mpan": 2,
          "meters": [
            {
              "active_from": {
                "__type": "<class 'datetime.date'>",
                "isoformat": "2025-06-23"
              },
              "active_to": null,
              "serial_number": 1,
              "is_export": false,
              "is_smart_meter": true,
              "device_id": "**REDACTED**",
              "manufacturer": "1063 - Landis and Gyr",
              "model": "00050204",
              "firmware": "38040602",
              "latest_consumption": "2025-09-30T00:00:00+00:00",
              "device": {
                "total_consumption": 953.052,
                "consumption": 0.0,
                "demand": 35.0,
                "start": "2025-10-01T15:00:00+00:00",
                "end": "2025-10-01T15:30:00+00:00"
              }
            }
          ],
```
At the physical meter, pressing A for a few seconds, then tapping it does get me to a 'Total Act Export' display that shows 218kWh which is the number the sensor was showing before it went Unknown.

Edit:  I've written to hello@.  Thank you for the help.

**Reply by BottlecapDave** [reply]
Looking at the diagnostic logic, it looks like theres a bug in reporting the export field. This bug is only present in diagnostics. This will be fixed in the next update.

**Reply by deltamelter** [reply]
> Looking at the diagnostic logic, it looks like theres a bug in reporting the export field. This bug is only present in diagnostics. This will be fixed in the next update.

Maybe a bug too, but the raw graphql API is reporting 0 for total exported.  The entity shows as unknown.  As somebody else reported, the meter still hold this data so either the mini changed what it gathers or OE changed what it stores/publishes on import meters.

**Reply by MPC-GH** [reply]
Phil at hello@ has come back to me, and asked for the Kraken API query being used to populate the 'Current Total Export Electricity' entity.  I didn't see a way to get to that from the home assistant interface/diagnostics and my google foo hasn't turned up anything.   Is anyone able to point me in the right direction?

Phil also said that my account doesn't have an export MPAN configured on it, and Octopus aren't able to access the export readings without that.  Thus the question about the query that is being used as he didn't think it could have been the actual meter export value (although it was the same value as far as I can tell).

The physical meter and the IHD both show the same value (218), and matched what was arriving in this entity in Home Assistant until the 27th from what I can see.

**Reply by BottlecapDave** [reply]
@deltamelter This behaviour matches the experience provided by the Home Pro and treats zero values as an unknown or invalid value. As the query being read represents "Total energy exported by the meter measured in Wh.", this should never be zero apart from when your meter has been freshly installed.

@MPC-GH The query can be found at https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/custom_components/octopus_energy/api_client/__init__.py#L116, with escaping for python formatting. This was based on investigation by @deltamelter (https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/discussions/940#discussioncomment-12937945) and is only available for meters that record both import and export values (as made clear in the docs https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/entities/electricity/#current-total-export). This property wasn't originally available when Home Mini support was added.

**Reply by alex-taylor** [reply]
So my 'Current Total Export Electricity' sensor did this on Tuesday last week, since when I've had no value coming back from it...until just before lunchtime today when it suddenly came back to life.  The value reported is as expected, so it's not at all clear what has happened here!  I suspect OE broke the API, and has now fixed it again...

**Reply by MPC-GH** [reply]
The reading came back for me at about 12.18 today too, via Dave's integration.  I've posted back to the forum.

Phil and the team look to have worked some magic and the integration's picked it up as soon as it was available again.

**Reply by BottlecapDave** [reply]
Thanks for the update
